aspnetcore/blazor/performance/developer-tools-profiling.md

```diff
@@ -48,8 +47,12 @@ Property | Default | Set value to&hellip; | Description
 `<WasmNativeDebugSymbols>` | `true` | `true` | Controls building with native debug symbols.
 `<WasmBuildNative>` | `false` | `true` | Controls building the native executable.
 
+Enabling profilers has negative size and performance impact, so don't publish an app for production with profilers enabled. In the following example, a condition is set on a property group section that only enables profiling when the app is built with `/p:ProfilingEnabled=true` (.NET CLI) or `<ProfilingEnabled>true</ProfilingEnabled>` in a Visual Studio publish profile.
```

Comment:
The other alternative is to enumerate all the features on commandline, when you want have it anabled.

Like `/p:WasmProfilers=browser`